modules/services/skhd/default.nix

```diff
@@ -1,46 +1,71 @@
-{ config, lib, pkgs, ... }:
-
-with lib;
-
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
 let
   cfg = config.services.skhd;
-in
 
+  inherit (lib) mkOption types;
+in
 {
-  options = {
-    services.skhd.enable = mkOption {
+  options.services.skhd = {
+    enable = mkOption {
       type = types.bool;
       default = false;
       description = "Whether to enable the skhd hotkey daemon.";
     };
 
-    services.skhd.package = mkOption {
+    package = mkOption {
       type = types.package;
       default = pkgs.skhd;
       description = "This option specifies the skhd package to use.";
     };
 
-    services.skhd.skhdConfig = mkOption {
+    skhdConfig = mkOption {
       type = types.lines;
       default = "";
       example = "alt + shift - r   :   chunkc quit";
       description = "Config to use for {file}`skhdrc`.";
     };
-  };
 
-  config = mkIf cfg.enable {
-    environment.systemPackages = [ cfg.package ];
+    errorLogFile = mkOption {
+      type = types.path;
+      default = "/var/tmp/skhd.error.log";
+      example = "/Users/khaneliman/Library/Logs/skhd.log";
+      description = "Path to the error log file.";
+    };
+
+    outLogFile = mkOption {
+      type = types.path;
+      default = "/var/tmp/skhd.out.log";
+      example = "/Users/khaneliman/Library/Logs/skhd.log";
+      description = "Path to the stdout log file.";
+    };
+  };
```

Comment:
I think these should also match upstream and should default to being under `/tmp`